controller/lib/core/controller.h

```diff
@@ -25,6 +25,10 @@ struct ControllerState {
   // therefore volume) to 0 at each breath boundary.
   Volume patient_volume;
   VolumetricFlow net_flow;
+
+  // Sequentially increasing index of the current breath among all breaths
+  // handled since controller startup.
+  int breath_id = 0;
```

Comment:
How many breaths do we expect to handle in one go?
Should we handle rollover properly or just make this unsigned and handle the rollover on GUI side?
Should we rather use explicit t_uint32 instead of int which is environment specific?

Reply:
Used uint32_t, thanks.